framework/tools/generators.py

```diff
@@ -14,3 +21,47 @@ def generate_string(length: int, additional_characters: list = None) -> str:
         result += additional_characters
 
     return "".join(result)
+
+
+def generate_user(password: str = DEFAULT_PASSWORD) -> dict:
+    """Generating a user with the specified password
+
+    Args:
+        password: password for user
+    """
+
+    faker = Faker()
+    hashed_password = bcrypt.hashpw(password.encode(), bcrypt.gensalt()).decode()
+    return {
+        "id": faker.uuid4(),
+        "email": faker.email(),
+        "first_name": faker.first_name(),
+        "last_name": faker.last_name(),
+        "password": password,
+        "hashed_password": hashed_password,
+    }
+
+
+def generate_jwt_token(email: str = '', expired: bool = False) -> str:
+    # Payload for the token
+    if not expired:
+        payload = {
+            "sub": email,
+            "iat": datetime.datetime.utcnow(),
+            "exp": datetime.datetime.utcnow() + datetime.timedelta(days=1)
+        }
+    else:
+        payload = {
+            "sub": email,
+            "iat": datetime.datetime.utcnow(),
+            "exp": datetime.datetime.utcnow() - datetime.timedelta(seconds=1)
+        }
+
+    # Secret key (base64 encoded)
```

Comment:
the code should be clear without comments. And comments, since they are used, are for completely incomprehensible constructions. It seems that everything is simple here. Why comments? =)

Reply:
Fixed.